app/src/testDebug/kotlin/org/cru/godtools/ui/languages/LanguageSettingsLayoutPaparazziTest.kt

```diff
@@ -29,8 +29,24 @@ class LanguageSettingsLayoutPaparazziTest(
         )
     )
 
+    private val varyingState = State(
+        appLanguage = Locale.FRENCH,
+        appLanguages = 1234,
+        downloadedLanguages = persistentListOf(
+            Language(Locale.ENGLISH, "English", false),
+            Language(Locale.FRENCH, "French - Forced", true),
+            Language(Locale.GERMAN, "German - Forced", true),
+            Language(Locale("es"), "Spanish", false)
+        )
+    )
+
     @Test
     fun `LanguageSettingsLayout()`() {
         snapshot { LanguageSettingsLayout(state) }
     }
+
+    @Test
+    fun `VaryingLanguageSettingsLayout()`() {
+        snapshot { LanguageSettingsLayout(varyingState) }
+    }
```

Comment:
I meant a brand new `LanguageNamePaparazziTest` that just directly uses the `LanguageName` composable.

Take a look at how I setup [LanguageDownloadStatusIndicatorPaparazziTest](https://github.com/CruGlobal/godtools-android/blob/77160b4eed3e8443a15f1a1259f7b3ab7a9feea0/app/src/testDebug/kotlin/org/cru/godtools/ui/languages/downloadable/LanguageDownloadStatusIndicatorPaparazziTest.kt)

Reply:
The overall Layout Paparazzi tests I'm doing a few sample screens of the varying state they can be in, but not trying to be exhaustive of the individual components on the screen. Then we can have some separate paparazzi tests to exercise the individual components.